src/touchaction.js

```diff
@@ -83,6 +83,19 @@ TouchAction.prototype = {
         var hasPanY = inStr(actions, TOUCH_ACTION_PAN_Y);
         var hasPanX = inStr(actions, TOUCH_ACTION_PAN_X);
 
+
+        if (hasNone) {
+            //do not prevent defaults if this is a tap gesture
+
+            var isTapPointer = input.pointers.length === 1;
+            var isTapMovement = input.distance === 0;
```

Comment:
Yes we can change this value to give it a little leeway. I will go ahead and change it to 2 to keep it consistent with the author's logic for recognizing Tap gestures. Look at the TapRecognizer function for an understanding of how the author intends to recognize Tap gestures.